### DISCUSSION TITLE
Error: The indicated resource does not exist

### DISCUSSION THREAD

**Discussion by Serkun123** [xiaoyifang/goldendict-ng#1359] [Q&A]
I get the error message in the title of this topic (it is more or less that meaning, I have translated it from Spanish). Then the program crashes. It is particularly recurrent in dictionary groups with a lot of dictionaries. Anyone has any idea how to solve this crash?

**Comment by xiaoyifang** [comment]
you can put the original spanish error message here. Does gd-ng throw this error?
and what is your gd-ng version?

**Comment by Serkun123** [comment]
The last version of GoldenDict is the file  6.6.0-GoldenDict.exe_windows-2019_20240104.zip 
This is the error message (attached)
![error message](https://github.com/xiaoyifang/goldendict-ng/assets/48263434/4ccd61c9-b7fa-4a4b-8bdb-b492dfb779f6)

**Comment by xiaoyifang** [comment]
What's the steps you are taking when throw the error message?

you can disable some dictionaries to narrow down to the dictionary which has caused this error.

**Comment by xiaoyifang** [comment]
you can try this release https://github.com/xiaoyifang/goldendict-ng/releases/tag/v23.12.08-alpha.11af19de
and upload gd_log.txt(in your configuration folder) file here .

**Comment by Serkun123** [comment]
I seem to have solve the problem. It seems that a group with many dictionaries will crash, so I created smaller groups such as English (1), English (2), and so on, and switch between them when I need more results on the looked-up word. Thxs!!!

**Comment by xiaoyifang** [comment]
https://github.com/xiaoyifang/goldendict-ng/pull/1360
